ios/FluentUI.Demo/FluentUI.Demo/Demos/NavigationControllerDemoController.swift

```diff
@@ -150,7 +151,24 @@ class NavigationControllerDemoController: DemoController {
 
 // MARK: - NavigationControllerDemoController: UIGestureRecognizerDelegate
 
-extension NavigationControllerDemoController: UIGestureRecognizerDelegate {
+extension NavigationControllerDemoController: UIGestureRecognizerDelegate, SearchBarDelegate {
+
+       func searchBarDidBeginEditing(_ searchBar: SearchBar) {
+               searchBar.progressSpinner.stopAnimating()
+       }
+
+       func searchBar(_ searchBar: SearchBar, didUpdateSearchText newSearchText: String?) {
+               // Do Nothing
```

Comment:
remove?

Reply:
can't be removed. It is not optional.

---
In reply to: [468818602](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r468818602) [](ancestors = 468818602)

Reply:
i meant the comment

Reply:
removed :)

---
In reply to: [470124917](https://github.com/microsoft/fluentui-apple/pull/174#discussion_r470124917) [](ancestors = 470124917)